app/assets/javascripts/models/beta/iteration.js

```diff
@@ -182,13 +218,9 @@ export const groupBySprints = (stories = [], project, initialSprintNumber = 1, p
     const firstSprintIndex = 0;
     const isFromSprintInProgress = !Story.isUnstarted(story);
 
-    sprints = handleSprintsOverflow(
-      project,
-      sprints,
-      Story.getPoints(story),
-      initialSprintNumber,
-      velocity
-    );
+    if (hasOverflow(story, velocity) && !isFromSprintInProgress) {
```

Comment:
"overflow" is a sprint characteristic, not a story one,  so a story can't _have_ overflow. What do you think about renaming this function to `causesOverflow` or something like that?

Reply:
Ok, I was thinking of the story had a "limit", but you are right, the "limit" is of sprint, no of story.